### PR TITLE
Task09 - ReactJs

### PR DESCRIPTION
**Pull request by nortega12**
# Pull Request
## Description
For Task 09 I created a React Project in the folder **"task09-reactjs"** with the purpose of creating the One Piece Website in React. Also I added scss and created the components I thought were necessary to separate the information of the home page.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Run the command **npm start** and automatically React will open a localhost:3000 window with the project running.

*Also run the **npm install** to install all the dependencies.

### CODE REVIEW

File: task09-reactjs/task09/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <span id="burger-icon" className="icon-align-left"></span>
+            <span id="burger-icon" className="icon-times"></span>
+          </button>
+          <div className="logo">
+            <img className="one-piece-logo" src={require("../images/one-piece-logo.png")} alt="One Piece Logo" />
+          </div>
+          <div className="collapse navbar-collapse" id="navbarTogglerDemo01">
+              <ul className="navbar-nav">
+                <li className="nav-item">
+                  <a className="nav-link active" href="/">Home<span class="sr-only">(current)</span></a>
```
Comment (PamelaSVegaCedeno) [inline task09-reactjs/task09/src/components/Header.js]: Avoid class in React please check all project and check this https://bobbyhadz.com/blog/react-invalid-dom-property-class 
image

File: task09-reactjs/task09/src/components/General-info.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+import { useEffect, useState } from 'react';
+import '../scss/Body.scss';
+   
+
+ function GeneralInfo() {
+    const [ info, setInfo ] = useState(null);
+    
+    const fetchOnePieceInfo = async () => {
+        const url = 'https://static.akurey.com/trainings/OnePieceInformation.json';
+        try{
```
Comment (PamelaSVegaCedeno) [inline task09-reactjs/task09/src/components/General-info.js]: Add a space between try and {

File: task09-reactjs/task09/src/components/General-info.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            </div>
+        </div>
+        </form>
+        <div className="main-photos-container">
+            <div className="characters-container">
+                <div className="titles-container">
+                    <p className="section-titles">Characters</p>
+                    <a href="/characters" className="see-all-titles">See all</a>
+                </div>
+                <div className="characters-photo-container" id="characters-photo-container">
```
Comment (PamelaSVegaCedeno) [inline task09-reactjs/task09/src/components/General-info.js:53]: I think this warning comes from this "maps"
image
Please check the warning ( https://reactjs.org/docs/lists-and-keys.html#keys)

File: task09-reactjs/task09/src/components/General-info.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                                <p className="mist-obj-title">{item.name}</p>
+                                <p className="mist-obj-text">{item.description}</p>
+                            </div>
+                        </div>
+                    )
+                   ))}
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (PamelaSVegaCedeno) [inline task09-reactjs/task09/src/components/General-info.js:111]: Please check the characters,islands and objects scroll specially in mobile version

**Review by PamelaSVegaCedeno — CHANGES_REQUESTED** [review]
Check the comments

File: task09-reactjs/task09/src/components/General-info.js (hunk 4 above)
Comment (nortega12) [reply task09-reactjs/task09/src/components/General-info.js:111]: Done, I added the scroll to all 3 :)

**Review by PamelaSVegaCedeno — APPROVED** [review]

**Pull request merged**